app-modules/core/src/Models/Role.php

```
@@ -2,6 +2,23 @@
+use Illuminate\Database\Eloquent\Casts\Attribute;
+use Metafori\Core\Enums\Role as RoleEnum;
 use Spatie\Permission\Models\Role as BaseRole;
 
-class Role extends BaseRole {}
+class Role extends BaseRole
+{
+    public function enum(): Attribute
+    {
+        return Attribute::get(
+            fn () => RoleEnum::from($this->name)
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Database\Eloquent\Casts\Attribute' in method 'enum'. (undefined)

(StaticAccess)